code/modules/mob/living/carbon/human/human_movement.dm

```diff
@@ -47,6 +47,12 @@
 
 	if(slowdown)
 		tally += 1
+	
+	if(l_hand.slowdown_hold)
```

Comment:
you can just remove the if's and just do tally += l_hand.slowdown_hold + r_hand.slowdown_hold , they're just needless bloat